utils/build_swift/build_swift/driver_arguments.py

```diff
@@ -494,6 +494,14 @@ def create_argument_parser():
                 'imply using lto on the swift standard library or runtime. '
                 'Options: thin, full. If no optional arg is provided, full is '
                 'chosen by default')
+    option('--darwin-enable-lto-only-for-cmake-targets', store,
+           default='',
+           help='A semi-colon split list of CMake targets for '
```

Comment:
We should mention that this only applies to swift target names, or else we should extend it to also work in llvm targets.  The normal build-script invocation is going to build executable and .dylib targets in llvm, and if you enable LTO it would be surprising that this option didn't affect those.